### DISCUSSION TITLE
Daikin FTXA50A2V1BW S21

### DISCUSSION THREAD

**Discussion by jaybeeking** [revk/xESP32-Faikout#425] [Q&A]
Hi,

I am trying to connect to a Daiking FTXA50A2V1BW maschine with an S21 cable.
On the PCB I can see an unused white S21 port. See image attached!

![IMG_3868](https://github.com/user-attachments/assets/ed1f7fb8-314c-478c-a6f4-bf73044ec118)

So I thought I could easily connect one of the official Daikin WiFi Controller Cabels to this port.
Unfortunately it does not fit in. It has other dimensions.

Are there different versions of S21 existing? How is the one on the PCB called and where can I buy a connector or entire cable?

![IMG_3873](https://github.com/user-attachments/assets/b05f854a-8ad2-41d6-be86-75d8b77833f9)

MANY THANKS for your help!

Juergen

**Comment by dahadd** [comment]
Hello, off-topic, but how did you remove the housing from this indoor unit? Thank you

**Reply by jaybeeking** [reply]
hi, i just followed the instructions from chapter 6.2 out of this [Manual](https://www.daikin.at/content/dam/document-library/Installer-reference-guide/ac/split/FTXA20-50AW,AT,AS_CTXA15AW,AT,AS_4PDE518023-3B_2018_06_Installer%20refernece%20guide_German.pdf#page12)

**Reply by dahadd** [reply]
thanks for the link 👍

**Comment by revk** [comment]
That plug looks like it fits, surely. Can you show you trying it?

I can go take a picture of one of mine maybe.

**Reply by jaybeeking** [reply]
I tried it serveral times. Look at the picture. The plug has a little bit different geometry.
The socket on the PCB also has room for a "clamp" but the plug does not have a clamp. See red arrwow on the screenshot.

A picture of yours would be great.
And you are not aware of different versions of S21 plugs?

![Screenshot 2024-08-11 101213](https://github.com/user-attachments/assets/5e872241-7f35-46d1-b626-6bd4f9ce5e54)

**Reply by revk** [reply]
Oh, that end, hmm, OK, not sure then.

**Reply by jaybeeking** [reply]
any idea whom I could ask here?

**Reply by revk** [reply]
Well, other users, to be honest. This module fits the wifi module end of the cable for S21. My understanding is that the PCB end is *usually* the same. If there are PCBs with different, I assume Daikin do a corresponding cable.

The other trick people have done is get jumper wire, i.e. with individual pin sockets on each end. 

![61XFclxlJ2L _SL1000_](https://github.com/user-attachments/assets/3b3f9eeb-ce4c-4aaa-94cb-2bb1d21fa4a3)

**Reply by jaybeeking** [reply]
Alright, I understand. The trick with jumper wire could work for me too. Many thanks!

Where can I find the documentation of ESP32-Faikin so that I know what Pin of the socket is what exactly?

On the indoor unit I can measure out GND and 12V power. But how do I find out Tx vs Rx?

**Reply by revk** [reply]
It's 1 to 1 pins. Easy!

**Reply by jaybeeking** [reply]
Ah, alright. Got it :-) Thanks for your support!

And by the way: I already have 3x FTXM20M2V1B working perfectly with the Faikin Controller. Great work done!

**Reply by jaybeeking** [reply]
I tried the jumper wires but they did not fit. Not enough space to connect all of them next to each other.

But I found a solution with this S21 Adapter cable:
![Screenshot 2024-09-04 192100](https://github.com/user-attachments/assets/4959305e-bf71-4e80-82aa-72b9b41a0d21)
Just search on Google for "Daikin EKRS21 S21-Steckplatzadapter".

Now everything works perfect!

**Comment by joealden** [comment]
@jaybeeking sorry off topic, but does your Stylish (FTXA) unit make a high pitched "buzzing" noise whenever the fan is running? I've contacted Daikin after my installer did the classic :shrug: so will see what they say, but wanted to know from an actual owner if this is normal or not - thanks! I've never heard any other air con unit make this noise!

P.S. I can record a video with audio too if needed to show what I mean - just let me know!

**Reply by jaybeeking** [reply]
hey, no there is no such kind of noise ... I guess your unit has some kind of defect :-(

**Reply by joealden** [reply]
Thanks for the response! That's kinda what I wanted to hear anyway so I can tell Daikin + my installer that I want it fixed!

**Comment by Ambass** [comment]
Hi,  Can you record it ? I think its' only yours. 
I've 8 of them, I do not notice high noise from the fan. Only when it starts, the brushless motor is a bit  ( little ! ) noisy.
Units are globally quiet during night ( in night mode , for sure ). 
The most annoying noise is the stepper motor of the horizontal swing and the poor temperature regulation quality ( solved in I2C sensor patching )

**Reply by joealden** [reply]
@Ambass thanks for the response! I've got a couple videos on https://www.reddit.com/r/hvacadvice/comments/1gqk14c/daikin_stylish_high_pitched_noise_expected/ (along with more detailed info) - let me know what you think!

>  Only when it starts, the brushless motor is a bit ( little ! ) noisy.

If the noise was just in startup I wouldn't mind - but this is constant!

> The most annoying noise is the stepper motor of the horizontal swing 

Yeah I noticed that too - I have swing turned off for now, but I feel like the noise of that motor is something I could tune out more easily (as it's a low hum not a high screech).

> and the poor temperature regulation quality ( solved in I2C sensor patching )

Yeah - exactly why I'm planning to install some faikins and hook up some BLE devices - wanting to get this noise sorted first though so that they don't blame it on me tinkering!

**Reply by Ambass** [reply]
Honestly, I hear nothing in your videos. Btw, I tried with a spectrum analyser on iPhone and it appears noise around 20kHz as soon as fan is running.
I think it's the switching frequency of the fan brushlesss motor driver. It's normally constant frequency and is not function of fan speed.
Try with spectrum analyser to check if you see the same thing. 20kHz is more or less out of human range, and if this is the problem, I'm afraid aboud Daikin answer..... if there is ! 
![IMG_6895](https://github.com/user-attachments/assets/4e2c16ab-09a7-42e5-9b1e-524bd1e0e9a7)

**Reply by joealden** [reply]
Thanks - appreciate the help! Yeah that must be it - I'm seeing a similar peak at ~20k, with potentially some resonance frequencies. I guess it must be that my young ears haven't been exposed to enough heavy metal music - what I wouldn't give to be a little more deaf :wink:. I do find it odd that Daikin would choose a switching frequency that close to 20kHz though - would it really have costed that much more money/efficiency to do that at 25kHz or above? I guess if I'm going to be installing the Faikins anyway I can better pinpoint where the noise is coming from with the outer casing off - as if it's from the PCB not the motor itself, I might be able to add some sound deadening - warranty be damned!

**Comment by adambonny** [comment]
Just for reference if anyone else comes accross this thread the most recent stylish units use the new style S21 connector. See wiki: https://github.com/revk/ESP32-Faikin/wiki/List-of-confirmed-working-air-con-units

If you dont fancy crimping your own cables I had success purchasing connectors and pre-crimped cables from RS. I simply soldered and shrink wrapped them together, the resulting cables were the right length to tuck the faikin away within the lower recess of the chassis.

Here's some photos including the required RS parts:
![IMG_8307](https://github.com/user-attachments/assets/f04140e8-2086-41bc-a103-7efa8ce502fd)
![IMG_8305](https://github.com/user-attachments/assets/c2d19935-4471-466c-9f7d-59f897c7ec6a)
![IMG_8304](https://github.com/user-attachments/assets/47aa0abc-fee8-4794-8f1f-23f46fb70c06)
![IMG_8306](https://github.com/user-attachments/assets/dae8a7c5-8c7e-4da0-88d1-fa5ccba22560)
